contracts/UBI.sol

```diff
@@ -270,6 +337,73 @@ contract UBI is Initializable {
   * @return The current balance including accrued Universal Basic Income of the user.
   **/
   function balanceOf(address _human) public view returns (uint256) {
-    return getAccruedValue(_human).add(balance[_human]);
-  }  
-}
+    return (balance[_human] + getAccruedValue(_human));
+  }
+
+  /* Stream Functions */
+  
+/* Stream Events */
+  /**
+  * @dev Emitted when the `_src` creates new stream for `_dst` 
+  * `_src` is address(0) for Primary stream
+  * `_drip` are UBI units moved into`_dst` stream.
+  */
+  event NewStream(address indexed _src, address indexed _dst, uint256 _drip);
+
+  /**
+  * @dev Emitted when the `_src` ~ `_dst` is stopped
+  * `_src` is address(0) for Primary stream
+  * `_drip` are UBI units moved out of `_dst` stream.
+  */
+  event EndStream(address indexed _src, address indexed _dst, uint256 _drip);
+  
+  /**
+  * @dev Emitted when the `_src`'s PoH is revoked `_dst`.
+  * `_src` is address(0) for Primary stream
+  * `_drip` UBI units per second revoked from `_dst` stream.
+  */
+  event Revoked(address indexed _src, address indexed _dst, uint256 _drip);
+  
+  /** @dev Start secondary UBI stream to any address
+   * @param _dst destination address
+   * @param _drip UBI units per second moved in stream. 
+   * <10% >1% of accruedPerSecond
```

Comment:
Any reason for 10%? Someone might want to stream 100% somewhere.

Reply:
There's actually 2 limits in streams,  max 5 outgoing streams each with min 1% ~max of 10% of primary UBI stream.

Future governance can change both value.. but allowing 100% in secondary streams isn't compatible with changing primary UBI accruing rate. I'd suggest max cap of <80%, <10 outgoing streams so it's easy on revoke list/gas and PoH users always have some % hodling.